### DISCUSSION TITLE
attr not compiling

### DISCUSSION THREAD

**Discussion by tundratim** [Homebrew/discussions#334] [Linux]
Trying to install attr from source to a non-standard location.  On both centos7 and debian docker instances, I see the following configuration error which is preventing compliation:

`configure:3314: gcc-5   -lintl conftest.c  >&5
/opt/TundraWare/tools/bin/ld: cannot find -lintl
`
This convinces the build the that c complier cannot create executables and the build fails.

Ideas?

**Comment by iMichka** [comment]
I opened a fix for this: https://github.com/Homebrew/homebrew-core/pull/67335

Once it is merged to homebrew-core the change will me ported to linux in the next 24h.

**Reply by tundratim** [reply]
OK thanks.  I'll try that change manually here.

**Comment by tundratim** [comment]
I did a quick sweep and there are a fair number of formula that have -lintil referenced in them.  Some of them are conditional on OS.mac, but many are not.

**Reply by iMichka** [reply]
Yes. On Mac they need to have `-lintl`. On linux we disable it. But this has been done case by case, when we have time.
